guide-configs/self-hosted/data-encryption.json

```diff
@@ -1,12 +1,12 @@
 {
     "file_name": "data-encryption.mdx",
     "id": "data-encryption",
-    "file_dir": "production-deployment/self-hosted-guide",
-    "slug": "/self-hosted-guide/data-encryption",
-    "title": "Encrypting application data",
-    "sidebar_label": "Data encryption",
-    "sidebar_position": 7,
-    "description": "This guide explains what you need to develop to use Temporal in your production environment.",
+    "file_dir": "production-deployment",
+    "slug": "/production-deployment/data-encryption",
```

Comment:
Let's make sure to add a redirect from the old slug to the new one here.

Reply:
I'm not sure if I'm understanding Docusaurus behavior right, but after having moved the page up in the hierarchy without renaming the slug, it appears to actually be redirecting automatically without a manual rule. Could use a head check on that though.

Reply:
nvm — caught the redirect and fixed.